capnp/src/lib.rs

```diff
@@ -25,6 +25,250 @@
 //! [Cap'n Proto](https://capnproto.org) messages in Rust. It is intended to
 //! be used in conjunction with code generated by the
 //! [capnpc-rust](https://github.com/capnproto/capnproto-rust/capnpc) crate.
+//!
+//! # Examples
+//!
+//! The following example shows how to send and process a simple RPC.
```

Comment:
Since this documentation is mainly about RPC, I don't think it belongs in the base `capnp` crate. `capnp-rpc` seems like a more appropriate place.